src/Setup/DevDivVsix/PortableFacades/PortableFacades.swr

```diff
@@ -1,25 +1,20 @@
 use vs
 
 package name=PortableFacades
-        version=1.0.0.0
+        version=1.1.0.0
```

Comment:
I mean, the automatic version update happens for components.json. 
Components who reference the facades should also pick it up without any manual modification as I have set them up to get any version within the range [1.0.0.0, 2.0.0.0).